.jenkins/common_job_properties.groovy

```diff
@@ -162,8 +162,11 @@ class common_job_properties {
   // Sets common config for Maven jobs.
```

Comment:
I agree with you, probably the wisest is to do this changes incrementally, so I will rebase this, so we can merge first the version changes, and then the other ones if you agree.